alicloud/resource_alicloud_instance.go

```diff
@@ -1,15 +1,17 @@
 package alicloud
 
 import (
+	"encoding/base64"
+	"encoding/json"
```

Comment:
Please remove useless code of line 4 and line 5.

Reply:
Already updated and I have been compiled locally.